UI/window-basic-settings.cpp

```diff
@@ -1705,6 +1718,15 @@ void OBSBasicSettings::LoadAdvOutputAudioSettings()
 	track5Bitrate = FindClosestAvailableAACBitrate(track5Bitrate);
 	track6Bitrate = FindClosestAvailableAACBitrate(track6Bitrate);
 
+	/* restrict list of bitrates  when multichannel is OFF*/
```

Comment:
Style:  Single line comments use `//`.  There is an extra space between "bitrates" and "when".